kolibri/plugins/default_theme/kolibri_plugin.py

```diff
@@ -61,7 +60,10 @@ def theme(self):
                 theme_hook.SHOW_K_FOOTER_LOGO: False,
             },
             # side-nav config
-            theme_hook.SIDE_NAV: {theme_hook.SHOW_K_FOOTER_LOGO: True},
+            theme_hook.SIDE_NAV: {
+                theme_hook.BRANDED_FOOTER: {},
```

Comment:
This looks fine - could you open an issue to update this: https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/utils/branding.js when this gets merged up to develop?

Reply:
https://github.com/learningequality/kolibri/issues/5968